Automattic/vip-go-mu-plugins
# Remove legacy jetpack submodule

## Description

The Jetpack plugin in `mu-plugins` is now based on versioned subtrees. This branch removes the legacy submodule, as we are no longer using it (or we should be no longer using it)

Concerns:
- If any code (especially on site repos) is directly requiring any Jetpack class from `mu-plugins/jetpack`, they will fail. To solve this, we could add a `jetpack` symlink pointing to the active versioned subtree.
- Removing a submodule is a painful process, as merges will fail for everyone in their local clone, and it will require manual intervention. Adding the symlink explained above would only exacerbate the problem. If anyone has any idea on how to do this without needing manual intervention, please comment.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
1. Ensure everything keeps working, especially Jetpack
